src/software/ai/navigator/path_planner/hrvo/hrvo_simulator.cpp

```diff
@@ -161,6 +259,18 @@ void HRVOSimulator::updateRobotVelocity(RobotId robot_id, const Vector &new_velo
     }
 }
 
+
+void HRVOSimulator::updateRobotAngularVelocity(
+    RobotId robot_id, const AngularVelocity &new_angular_velocity)
+{
+    auto hrvo_agent = robots.find(robot_id);
+    if (hrvo_agent != robots.end())
+    {
+        hrvo_agent->second->setAngularVelocity(new_angular_velocity);
+    }
```

Comment:
I decided against logging since the log would get spammed when ever the robot is on but not visible in vision (i.e. the Agent doesnt exist in simulator, and the log is spammed at every tick)